pkg/daemon/ceph/mds/filesystem.go

```diff
@@ -100,6 +101,10 @@ func (f *Filesystem) CreateFilesystem(context *clusterd.Context, clusterName str
 
 	// create the file system
 	if err := client.CreateFilesystem(context, clusterName, f.Name, f.metadataPool.Name, dataPoolNames, f.activeMDSCount); err != nil {
```

Comment:
I think this function is called on every start of the operator.
What happens if there is an issue ("bad" exit code but not fatal) for an existing Pool? This would delete the existing Pool.